antidox/tester.py

```diff
@@ -0,0 +1,47 @@
+import requests
+import json
+import urllib.parse
+import time
+import clean
+import datetime
+
+
+def wiki_write(result, header):
+    site = pywikibot.Site()
+    repo = site.data_repository()
+    page = pywikibot.Page(site, u"User_talk:" + args.wikibot)
+
+    heading = (header)
+    content = (result)
+    message = "\n\n{}\n{} --~~~~".format(heading, content)
+    page.save(summary="Testing", watch=None, minor=False, botflag=True,
+              force=False, async=False, callback=None,
+              apply_cosmetic_changes=None, appendtext=message)
+
+def log_change():
+  while True:
+    start = datetime.datetime.now() - datetime.timedelta(minutes=2)
+    rcstart = start.isoformat()
+    end = datetime.datetime.now()
```

Comment:
`datetime.datetime.now()` gives the time in the current local timezone, but Mediawiki expects UTC timestamps. (This is probably why you didn't get results before.) Just change `datetime.datetime.now()` to `datetime.datetime.utcnow()` to get the time in UTC.